testing/fixtures_base.h

```diff
@@ -0,0 +1,49 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_TESTING_FIXTURES_BASE_H_
+#define FLUTTER_TESTING_FIXTURES_BASE_H_
+
+#include <memory>
+
+#include "flutter/common/settings.h"
+#include "flutter/runtime/dart_vm.h"
+#include "flutter/testing/elf_loader.h"
+#include "flutter/testing/test_dart_native_resolver.h"
+#include "flutter/testing/testing.h"
+#include "flutter/testing/thread_test.h"
+
+namespace flutter::testing {
+
+class FixturesBase {
```

Comment:
Can we call this something like `DartFixture` or something? Since this fixture seems to be primarily dealing with setting up the VM and isolates in various modes? Perhaps also add a docstring.